test/brpc_http_rpc_protocol_unittest.cpp

```diff
@@ -310,6 +325,12 @@ TEST_F(HttpTest, verify_request) {
         VerifyMessage(msg, false);
         msg->Destroy();
     }
+    {
+        brpc::policy::HttpContext* msg =
+                MakePostProtoTextRequestMessage("/EchoService/Echo");
+        VerifyMessage(msg, false);
+        msg->Destroy();
```

Comment:
I seems that this unit test does not cover any code you change in `http_rpc_protocol.cpp`. You can reference the unit test named `spring_protobuf_content_type` in `brpc_http_rpc_protocol_unittest.cpp` which also test a new content type.

Reply:
add new spring_protobuf_text_content_type ut.

Reply:
Sorry, forgot to push the commit. Now it should be there.

Reply:
Good job!